src/applications/static-pages/wizard/index.js

```diff
@@ -171,8 +171,11 @@ Wizard.propTypes = {
   pages: PropTypes.arrayOf(
     PropTypes.shape({
       name: PropTypes.string.isRequired,
-      component: PropTypes.oneOfType([PropTypes.func, PropTypes.node])
-        .isRequired,
+      component: PropTypes.oneOfType([
+        PropTypes.elementType,
```

Comment:
adding `PropTypes.elementType` allows redux connect to be used with React components when passed to the wizard... ex
```
export default {
  name: PAGE_NAMES.recipients,
  component: connect(mapStateToProps)(Recipients),
};
```